api/bucketsd/service.go

```diff
@@ -91,6 +91,7 @@ var (
 		DealMinDuration: powUtil.MinDealDuration,
 		FastRetrieval:   true,
 		DealStartOffset: 72 * 60 * 60 / powUtil.EpochDurationSeconds, // 72hs
+		MaxPrice:        100_000_000_000,
```

Comment:
Let this default storage-config have some safer MaxPrice instead of 0 (which is... no max price).
The user always can update it to anything it wants.